internal/js/modules/k6/browser/tests/static/react_input.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>Controlled Grafana-like Input Demo</title>
+  <!-- React and ReactDOM via UNPKG -->
+  <script crossorigin src="https://unpkg.com/react@18/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
```

Comment:
In general it is not great when our test depends on other services being up and running and we have tried to remove those.

https://github.com/grafana/k6/issues/464 https://github.com/grafana/k6/issues/2302

Reply:
I guess a "quick-win" could be copy-pasting those files locally, right? 🤔 So, this way we don't rely on an external server, to be available. Reproducing the issue (if React-specific) with a subset of the library might more challenging, tho.

Reply:
Instead of saving these scripts in the repository and bloating it, why don't we download them once, right before the test starts, if the files don't exist?